tests/installation/multipath.pm

```diff
@@ -19,7 +19,19 @@ use testapi;
 sub run {
     # Sometimes multipath detection takes longer
     assert_screen "enable-multipath", 60;
-    send_key "alt-y";
+    # This module is supposed to be invoked only if MULTIPATH is true
+    # or MULTIPATH_DIALOG_YES is set
+    if (get_var("MULTIPATH") or get_var("MULTIPATH_DIALOG_YES")) {
+        send_key "alt-y";
+    }
+    else {
+        # MULTIPATH_DIALOG_YES set to false
+        send_key "alt-n";
+    }
+    # For the record: report on the text console
+    send_key "ctrl-alt-f9";
+    assert_script_run "cat /proc/partitions; echo 'Report: multipath -ll'; multipath -ll";
+    send_key "ctrl-alt-f7";
```

Comment:
Not sure if this check is really necessary.

Reply:
Well, the resulting multipath setup should be reported somewhere. Preferably in the logs, of course. I did not find anything logged without this check, so I put it in here for simplicity. I agree that the check is sort of preliminary, but it sure won't hurt. Any hints on how to manage to get a good log entry instead?

Reply:
Installation logs are already collected  file `udev.log`  contains details about devices during installation.  I would prefer to do muttipath checks in separate module  after first boot.

Reply:
OK, took this check out again. As you say, udev.log does tell.